__tests__/config.test.ts

```diff
@@ -26,6 +26,12 @@ test('it reads custom configs', async () => {
   expect(config.allow_licenses).toEqual(['BSD', 'GPL 2'])
 })
 
+test('it defaults to false for warn-only', async () => {
+  setInput('warn-only', 'false')
```

Comment:
If this is meant to test the default, is it really necessary to have this line?

Reply:
Not necessary, removing it